docs/testing.rst

```diff
@@ -47,7 +47,8 @@ To run all tests under certain python and django versions:
 
 .. code-block:: bash
 
-    $ tox -e py27-django10          # run all tests under python 2.7 and django 1.10
+    $ tox -e py27-django110         # run all tests under python 2.7 and django 1.10
+    $ tox -e py35-django111         # run all tests under python 3.5 and django 1.11
```

Comment:
It was previously incorrect, and thought an extra example with py35 would be nice.